prog/dftb+/lib_dftb/periodic.F90

```diff
@@ -512,12 +528,12 @@ function getNrOfNeighbours(neigh, cutoff, iAtom) result(nNeighbourSK)
     integer, intent(in) :: iAtom
 
     !> Nr. of neighbours for the specified atom.
-    integer :: nNeighbourSK
+    integer :: nNeighbour
```

Comment:
Is this SK cut-off neighbours? Might be better to be consistent with naming elsewhere.